src/VisualStudio/Core/Def/ServicesVSResources.resx

```diff
@@ -1024,6 +1024,18 @@ I agree to all of the foregoing:</value>
   <data name="Decompiler_Legal_Notice_Title" xml:space="preserve">
     <value>Decompiler Legal Notice</value>
   </data>
+  <data name="Colorize_regular_expressions" xml:space="preserve">
```

Comment:
>Colorize_regular_expressions [](start = 14, length = 28)

Would you mind adding some screenshots to the PR, to show the context those strings appear in? Thanks

Reply:
I don't think we use the word "Colorize" elsewhere publicly, but I'm not sure what would be better. Do customers know what "Classify" means instead? Or "use fonts and colors settings" (since it will also bold)? Not sure what to do here. Opinions @dotnet/roslyn-ide?